main.go

```diff
@@ -76,3 +77,27 @@ func main() {
 
 	os.Exit(0)
 }
+
+// TODO: This is unix system specific. When we get around to supporting windows we will want to
+// make this more platform agnostic.
+func getWrappedCommand() (cmdPath string) {
+	// Set this variable to the path of the cheque wrapper link
+	var chequePath = ""
+
+	var paths = strings.Split(os.Getenv("PATH"), ":")
```

Comment:
Instead of `strings.Split()` I think you can use `filepath.SplitList()`

Direct from the `LookPath` impl:

```
	path := os.Getenv("PATH")
	for _, dir := range filepath.SplitList(path) {
		if dir == "" {
			// Unix shell semantics: path element "" means "."
			dir = "."
		}
		path := filepath.Join(dir, file)
		if err := findExecutable(path); err == nil {
			return path, nil
		}
	}
```

Reply:
Fixed. Also now use filepath.Join